### DISCUSSION TITLE
NodeRED Error in Funktion für Timeframe-Node

### DISCUSSION THREAD

**Discussion by electric-m** [camueller/SmartApplianceEnabler#413] [Allgemein]
Hallo,

ist mir nur zufällig aufgefallen, dass hier zwei Fehler ausgegeben werden.
Ich habe einfach nur mal die Basis-Flows übernommen, bisher noch  nichts angepasst.
Es betrifft die UI-Node Timeframes. Irgendwo klemmt die Funktion davor:

`Incorrect timeFormat value :"undefined". This timeFormat was corrected with the default value: "%Y-%m-%d %H:%M:%S".`

und

`TypeError: Cannot read properties of undefined (reading 'addDays')`

Viele Grüße

**Comment by camueller** [comment]
> TypeError: Cannot read properties of undefined (reading 'addDays')

Da ist wohl DateFNS nicht installiert entsprechend der Anleitung.

Möglicherweise fehlen die anderen Bibliotheken auch und führen zu dem anderen Fehler.
